Autopilot/PathManager/Src/pathStateClasses.cpp

```diff
@@ -94,6 +94,40 @@ void commsWithTelemetry::execute(pathManager* pathMgr)
 {
     GetTelemetryCommands(&_incomingData);
 
+    // incoming angles are radians
+    static float gimbalMaxPitch = 3.14159;
+    static float gimbalMaxYaw = 3.14159/2;
```

Comment:
you could make these `const`

Reply:
am I allowed to use the c++ math library?